packages/base/src/panelview/annotationPanel.tsx

```diff
@@ -20,14 +20,14 @@ export class AnnotationsPanel extends Component<IAnnotationPanelProps> {
     this._annotationModel = props.annotationModel;
     this._rightPanelModel = props.rightPanelModel;
 
-    this._annotationModel.contextChanged.connect(async () => {
-      await this._annotationModel?.context?.ready;
+    this._annotationModel.modelChanged.connect(async () => {
+      // await this._annotationModel?.context?.ready;
```

Comment:
@gjmooney do you remember if this await was required ? Should we restore something similar, now we removed the context ?